src/main/java/com/alkemy/ong/domain/slides/SlidesService.java

```diff
@@ -0,0 +1,19 @@
+package com.alkemy.ong.domain.slides;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class SlidesService {
+
+    private final SlidesGateway slidesGateway;
+
+    public SlidesService(SlidesGateway slidesGateway) {
+        this.slidesGateway = slidesGateway;
+    }
+
+    public List<Slides> findAll(){
+        return this.slidesGateway.findAll();
```

Comment:
el this no es necesario. Se usa en los constructores para diferenciar el nombre de los param de entrada con el de los atributos de la clase cuando son iguales.

Reply:
Fixed